.github/workflows/ruby.yml

```diff
@@ -85,10 +85,56 @@ jobs:
           }
 
       - name: test
-        timeout-minutes: 10
+        timeout-minutes: 8
         run:  |
           # RubyGems < 3, no frozen strings
           if ('${{ matrix.ruby }}' -ge '2.6') {
             $env:RUBYOPT = '--enable=frozen-string-literal'
           }
           bundle exec rake
+
+  nonMRI:
+    name: >-
+      ${{ matrix.cfg.os }}, ${{ matrix.cfg.ruby }}
+    env:
+      CI: true
+      TESTOPTS: -v
+
+    runs-on: ${{ matrix.cfg.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        cfg:
+          - { os: ubuntu-latest, ruby: jruby-9.2        }
```

Comment:
JRuby 9.2 failure should not be allowed - that build passes and is one of our supported platforms.

Reply:
Personally I'd just put it in the main action and call this one nonMRIHead or something

Reply:
Done